sklearn/utils/estimator_checks.py

```diff
@@ -1,11 +1,20 @@
 from __future__ import print_function
 
+import os
 import types
 import warnings
 import sys
 import traceback
 import inspect
 import pickle
+import tempfile
+import shutil
+
+try:
+    WindowsError
+except NameError:
+    WindowsError = None
```

Comment:
Maybe add a comment here to make it explicit that this exception is only defined and can only be raised on the windows platform but we need to make the code run everywhere.